javascript/javascript3/week3/user.js

```diff
@@ -0,0 +1,233 @@
+class Users {
+    constructor() {
+        this.form = document.querySelector('form');
+        this.formLog = document.querySelector('.formLog');
+        this.email = document.querySelector('[type="email"]');
+        this.password = document.querySelector('.password');
+        this.emailLogIn = document.querySelector('.emailLogIn');
+        this.passwordLogIn = document.querySelector('.passwordLogIn');
+        this.inputs = document.querySelectorAll('input');
+        this.fullCollection = document.querySelector('.fullCollection');
+        this.deleteInput = document.querySelector('.deleteInputUser'); 
+        this.url = 'https://crudcrud.com/api/ce64f626ce31465592a412be09c55354/users';
+    }
+
+    clearInputs() {
+        this.inputs.forEach(item => {
+            item.value = '';
+        }); 
+    }
+
+    checkMailInputs(){
+        this.email.addEventListener('keypress', function(e) {
```

Comment:
hmmm... what does that do? You can think about how to refactor this method so it is more obvious what is the purpose of that. Does it only allow to type in certain characters into the email input? That is bit risky. I am not sure waht carachters are allowed in email and waht are not but there might be some crazy letters, like danish ones, that are allowed, or special characters. Here is some more info. https://stackoverflow.com/questions/2049502/what-characters-are-allowed-in-an-email-address
If you need to validate this input you can use the native browser validation -